apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/mapEntryAssertions.kt

```diff
@@ -22,6 +22,9 @@ infix fun <K, V, T : Map.Entry<K, V>> Expect<T>.isKeyValue(keyValuePair: Pair<K,
  * Creates an [Expect] for the property [Map.Entry.key] of the subject of `this` expectation,
  * so that further fluent calls are assertions about it.
  *
+ * @sample ch.tutteli.atrium.api.infix.en_GB.samples.MapEntryExpectationSamples.keyFeature
+ * @sample ch.tutteli.atrium.api.infix.en_GB.samples.deprecated.MapEntryAssertionSamples.keyFeature
```

Comment:
I think you got me wrong. Sorry about that. There is no need to point to the deprecated sample where we already have the new one
```suggestion
```
But you can add a `@sample` to the `isKeyValue` method above:
```
@sample ch.tutteli.atrium.api.infix.en_GB.samples.deprecated.MapEntryAssertionSamples.isKeyValue
```

Unfortunately I cannot suggest code changes for code you have not modified, otherwise I would do it myself